parsers/ENTSOE.py

```diff
@@ -857,27 +806,30 @@ def fetch_production(zone_key, session=None, target_datetime=None,
 
     data = []
     for i in range(len(production_dates)):
-        production_values = {ENTSOE_PARAMETER_DESC[k]: v for k, v in
-                             productions[i].items()}
+        production_values = {k: v for k, v in productions[i].items()}
         production_date = production_dates[i]
 
+        production_types = {'production': {}, 'storage': {}}
+        for key in ['production', 'storage']:
+            parameter_groups = ENTSOE_PARAMETER_GROUPS[key]
+            multiplier = -1 if key == 'storage' else 1
+
+            for fuel, groups in parameter_groups.items():
+                has_value = any([grp in production_values.keys() for grp in groups])
```

Comment:
Here only the existence of the entry is checked for, but not the value itself.
I suggest
```suggestion
                has_value = any([production_values.get(grp) is not None for grp in groups])
```
As `production_values.get(grp)` would give None if a key is missing or if the value is None